includes/classes/DistributorPost.php

```diff
@@ -816,6 +817,38 @@ protected function to_insert( $args = array() ) {
 		return $insert;
 	}
 
+	/**
```

Comment:
@peterwilsoncc Can we add `since` tag?

Reply:
I've created #1051 to add the annotations as I noticed I missed them for all the methods in the new class. I've added it to this method as it's introduced in this PR, see 671837f4e402c9457710c9b6a20e6a20b1598f43